src/api/app/models/staging/staged_requests.rb

```diff
@@ -29,7 +29,7 @@ def destroy
     not_deleted_packages = package_names - result.pluck(:name)
 
     requests.each do |request|
-      add_review_for_unstaged_request(request)
+      add_review_for_unstaged_request(request) if [:review, :new].include?(request.state.to_sym)
```

Comment:
there is tons of code like `req.state.in?([:review, :new])` and `@bs_request.state.in?([:new, :review])` so I don't think you need .to_sym, but should prefer `.in?` for consistency.

And btw, I found this gem:
```
app/models/bs_request.rb:    unless state == :review || (state == :new && state == :new)
```

Reply:
> there is tons of code like `req.state.in?([:review, :new])` and `@bs_request.state.in?([:new, :review])` so I don't think you need .to_sym, but should prefer `.in?` for consistency.

Yup, i will do it like that, thanks!

> And btw, I found this gem:
> 
> ```
> app/models/bs_request.rb:    unless state == :review || (state == :new && state == :new)
> ```
:pray: TGIF :pray:

Reply:
done @coolo